data-es2016plus.js

```diff
@@ -6993,6 +6993,118 @@ exports.tests = [
       rhino1_7_15: true
     }
   },
+  {
+    name: 'Change Array by copy',
+    category: '2023 features',
+    significance: 'small',
+    spec: 'https://github.com/tc39/proposal-change-array-by-copy',
+    subtests: [
+      {
+        name: "Array.prototype.toReversed",
+        exec: function () {/*
+          var arr = [1, 2, 3];
+          return arr.toReversed()[0] === 3 && arr[0] === 1;
+        */},
+        res: {
+          babel7corejs3: babel.corejs,
+          typescript3_2corejs3: typescript.corejs,
+          ie11: false,
+          chrome109: false,
+          chrome110: true,
+          edge109: false,
+          edge110: true,
+          firefox114: false,
+          firefox115: true,
+          firefox102: false,
+          opera95: false,
```

Comment:
Recent Opera results are mirrored from chrome, no need to add it here.

Reply:
I have removed the opera browser in commit [8e40474](https://github.com/compat-table/compat-table/pull/1907/commits/8e40474707acbcea4b396e922eca44df2e9b0ccb).